eflowautodeploy/eflowAutoDeploy.ps1

```diff
@@ -493,6 +493,7 @@ function Test-EadEflowInstall {
     Write-Host "AzureEFLOW Module:$version"
     $svc = Get-Service wssdagent
     if ($svc.Status -ne 'Running') {
+        Set-Service -Name wssdagent -StartupType Automatic

```

Comment:
Why are we adding the StartupType Automatic?

Reply:
removed this completely